LICENCE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2011-2021, Dr David Ham, Chris Pickel and others
+
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without modification,
+are permitted provided that the following conditions are met:
+
+    * Redistributions of source code must retain the above copyright notice,
+      this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright notice,
+      this list of conditions and the following disclaimer in the documentation
+      and/or other materials provided with the distribution.
+    * Neither the name of {{ project }} nor the names of its contributors
```

Comment:
I guess we need `sphinx-contrib/youtube` in here instead of `{{project}}

Reply:
already fixed

forgot to comment